autoload/jedi.vim

```diff
@@ -489,6 +489,8 @@ function! jedi#complete_opened(is_popup_on_dot)
         " option is set.
         if g:jedi#popup_select_first && stridx(&completeopt, 'longest') > -1
             return "\<Down>"
+        else
+            return ""
```

Comment:
Have you tested it using the popup_on_dot feature and without "noinsert" or "noselect" in &completeopt?
That's when it's expected to return `<c-p>`, but would return `""` now.

Adding a test for these combinations would be really nice btw.. :)